src/Symfony/Component/AssetMapper/AssetMapperCompiler.php

```diff
@@ -22,23 +22,34 @@
  */
 class AssetMapperCompiler
 {
+    private AssetMapperInterface $assetMapper;
+
     /**
      * @param iterable<AssetCompilerInterface> $assetCompilers
```

Comment:
I suggest adding `@param \Closure(): AssetMapperInterface $assetMapperFactory` to document the expected signature of the closure